airbyte-workers/src/main/resources/kube_runner_template.yaml

```diff
@@ -8,14 +8,25 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: worker-container
+        - name: worker-run
           image: TAGGED_IMAGE
           workingDir: WORKDIR
           args: ARGS
           volumeMounts:
-            - name: airbyte-volume-workspace
-              mountPath: /workspace
+            - name: workdir
+              mountPath: WORKDIR
+      initContainers:
+        - name: worker-init
+          image: library/busybox:1.31
+          command:
+            [
+              "sh",
+              "-c",
+              "until [ -f WORKDIR/ready ]; do echo waiting...; sleep 1; done",
```

Comment:
We should have some sort of timeout here.

Reply:
I'll add that. We should also have timeouts in `kube_runner.sh` too.